test/controllers/hosts_controller_test.rb

```diff
@@ -449,8 +456,8 @@ def setup_multiple_environments
     post :update_multiple_environment, params,
       set_session_user.merge(:user => users(:admin).id)
 
-    assert Host.find(@host1.id).environment == hostgroup.environment
-    assert Host.find(@host2.id).environment == hostgroup.environment
+    assert_equal hostgroup.environment_id, Host.unscoped.find(@host1.id).environment_id
```

Comment:
This is the test that broke in the last update, `.unscoped` was missing and also I compare now with `_id` instead of calling `.reload`.